### ISSUE TITLE
specific claim is showing wrong NSFW property

### ISSUE THREAD

**Issue by tiger5226** [lbryio/chainquery#45]
https://chainquery.lbry.io/api/sql?query=SELECT%20*%20FROM%20claim%20WHERE%20claim_id=%22cb132b147ab1e756dab4ccc9e3514716497edf2b%22

This claim shows isNSFW as false. However, looking in the lbry app it shows as NSFW. Why? There must be a bug in the sync process somewhere. This is important to look into because there is up and coming integration work that will rely on this property.

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of the user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by kauffj** [comment]
```
 ./lbrynet-cli resolve --uri="lbry://six#cb132b147ab1e756dab4ccc9e3514716497edf2b"
{
  "lbry://six#cb132b147ab1e756dab4ccc9e3514716497edf2b": {
    "certificate": {
      "address": "bUePKSg3RvEJcDc4HMHi54Rkztx4STt9Sp",
      "amount": 10.0,
      "claim_id": "9377895520f71f82702b2dc56cc45d2f046ada8a",
      "claim_sequence": 3,
      "decoded_claim": true,
      "depth": 131039,
      "effective_amount": 10.0,
      "has_signature": false,
      "height": 276808,
      "hex": "08011002225e0801100322583056301006072a8648ce3d020106052b8104000a03420004f533d37651c3f7e039061411d2a047835125cc2f2e42dbfc005054be35137654da8c6a7dafb277259e15b8a6f827c5deccada427735608b18d0346df78649d86",
      "name": "@Porn",
      "nout": 0,
      "permanent_url": "@Porn#9377895520f71f82702b2dc56cc45d2f046ada8a",
      "supports": [],
      "txid": "b80716bc6d06990e1210ff2af30f667703e964b0b23c9a112a3af8c191636553",
      "valid_at_height": 280324,
      "value": {
        "certificate": {
          "keyType": "SECP256k1",
          "publicKey": "3056301006072a8648ce3d020106052b8104000a03420004f533d37651c3f7e039061411d2a047835125cc2f2e42dbfc005054be35137654da8c6a7dafb277259e15b8a6f827c5deccada427735608b18d0346df78649d86",
          "version": "_0_0_1"
        },
        "claimType": "certificateType",
        "version": "_0_0_1"
      }
    },
    "claim": {
      "address": "bWmtPsF3Mgh9V1b3h9GfzCfvAYGNkDoToT",
      "amount": 20000.0,
      "channel_name": "@Porn",
      "claim_id": "cb132b147ab1e756dab4ccc9e3514716497edf2b",
      "claim_sequence": 6,
      "decoded_claim": true,
      "depth": 881,
      "effective_amount": 20000.0,
      "has_signature": true,
      "height": 406966,
      "hex": "080110011ad5010801128d01080410011a30576f6d616e2062727574616c6c7920666973746564207768696c6520696e20626f6e646167652036206d696e7574657322002a0032044e6f6e6538014224080110011a1955c5f0e5bce2d91ff9d5b7472ccfe12949d3ccb29a77d38fbe25000000404a1f68747470733a2f2f692e696d6775722e636f6d2f433037434c70482e706e6752005a001a41080110011a301a123aa303ee1f3376665822b9475954dd0a05e9b980b7b9404ba9013574ab97dd0d478de0d4fe4f1a144e8ad11712592209766964656f2f6d70342a5c080110031a402215d797faea8bf485ce2fe40f75b07fe70f68af3e487cddd68e2d49a8e87de9a6c94636e2b76e9081c673e705bfab0602e1f9f68a8808996cbdfb3fd0245bc422149377895520f71f82702b2dc56cc45d2f046ada8a",
      "name": "six",
      "nout": 0,
      "permanent_url": "@Porn#9377895520f71f82702b2dc56cc45d2f046ada8a/six",
      "signature_is_valid": true,
      "supports": [],
      "txid": "aac923c7be984fca0b6970d7bc4d0bc0aa2496e96c3fc9e41dadaeabbe61a020",
      "valid_at_height": 406976,
      "value": {
        "claimType": "streamType",
        "publisherSignature": {
          "certificateId": "9377895520f71f82702b2dc56cc45d2f046ada8a",
          "signature": "2215d797faea8bf485ce2fe40f75b07fe70f68af3e487cddd68e2d49a8e87de9a6c94636e2b76e9081c673e705bfab0602e1f9f68a8808996cbdfb3fd0245bc4",
          "signatureType": "SECP256k1",
          "version": "_0_0_1"
        },
        "stream": {
          "metadata": {
            "author": "",
            "description": "",
            "fee": {
              "address": "bWmtPsF3Mgh9V1b3h9GfzCfvAYGNkDoToT",
              "amount": 2.0,
              "currency": "LBC",
              "version": "_0_0_1"
            },
            "language": "en",
            "license": "None",
            "licenseUrl": "",
            *"nsfw": true,*
            "preview": "",
            "thumbnail": "https://i.imgur.com/C07CLpH.png",
            "title": "Woman brutally fisted while in bondage 6 minutes",
            "version": "_0_1_0"
          },
          "source": {
            "contentType": "video/mp4",
            "source": "1a123aa303ee1f3376665822b9475954dd0a05e9b980b7b9404ba9013574ab97dd0d478de0d4fe4f1a144e8ad1171259",
            "sourceType": "lbry_sd_hash",
            "version": "_0_0_1"
          },
          "version": "_0_0_1"
        },
        "version": "_0_0_1"
      }
    },
    "claims_in_channel": 0
  }
}
```

**Comment by jessopb** [comment]
The metadata license field is also useful/required.

**Comment by tzarebczan** [comment]
Could be related to https://github.com/lbryio/chainquery/issues/75 . Will open a new issue for license. Will double check if we are missing anything else currently (besides new fields we are adding).

**Comment by tiger5226** [comment]
Yeah, any missing fields just add here. It is the same issue. This can all be solved in one issue.

**Comment by kauffj** [comment]
@tiger5226 are you manually creating fields to match metadata?

**Comment by tiger5226** [comment]
solved with 1.5.0 release
